bloom/generators/debian/templates/rules.em

```diff
@@ -59,4 +60,7 @@ override_dh_auto_install:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_install
+	dh_auto_install --destdir=$(CURDIR)/debian/@(Package)
```

Comment:
Thanks William, yes it would be great if we could test it out on the farm to see how well it fares. I came across this Debian wiki that partially describes this approach:
https://wiki.debian.org/DebugPackage

Please see the section titled "Debhelper >= 7"